app/controllers/step_by_step_pages_controller.rb

```diff
@@ -97,6 +97,7 @@ def internal_change_notes
 
   def discard_draft
     StepNavPublisher.discard_draft(@step_by_step_page)
+    @step_by_step_page.discard_notes
```

Comment:
We're missing feature tests for deleting step by step pages. Can you add some?

Reply:
Should I make a new PR for that, or combine it with this one?

Reply:
Can you add it as a preparatory commit to this one? Cheers.

Reply:
As we discussed it's turned out to be a bit thornier than expected. I'll move it to its own PR and ticket.

Reply:
I think you need to remove this function call from here. As discussed at stand_up, we can instead call `discard_notes` from a `before_destroy` block in the StepByStepPages model, which we're going to tackle as part of the bug card.